one-pager/banner/main.py

```
@@ -0,0 +1,71 @@
+@functions_framework.http
+def generate_image(request):
+    """HTTP Cloud Function that generates an image using Gemini and uploads it to GCS."""
+
+    # Parse the request body for the prompt
+    try:
+        request_json = request.get_json(silent=True)
+        if request_json and 'prompt' in request_json:
+            prompt = request_json['prompt']
+        else:
+            return json.dumps({"error": "Missing 'prompt' in request body."}), 400
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)